HW03/anastasia_evsihneyeva/spec/anastasia_evsihneyeva_hw_03_t_01_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative '/home/anastasia/Документы/ruby/homeworks-2022/spec/spec_helper'
```

Comment:
here and and in the all spec please use the correct relative path for the files you need